github/repos_collaborators.go

```diff
@@ -23,6 +23,13 @@ type ListCollaboratorsOptions struct {
 	// Default value is "all".
 	Affiliation string `url:"affiliation,omitempty"`
 
+	// Permission specifies how collaborators should be filtered by the permissions they have on the repository.
+	// Possible values are:
+	// pull, triage, push, maintain, admin
```

Comment:
For consistency, let's please put these in double quotes.

```suggestion
	// "pull", "triage", "push", "maintain", "admin"
```